test/integration/test_global_catalog_v1.py

```diff
@@ -0,0 +1,289 @@
+# -*- coding: utf-8 -*-
+# (C) Copyright IBM Corp. 2020.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Test the platform service Global Catalog API operations
+"""
+
+import unittest
+import os
+from ibm_platform_services import GlobalCatalogV1
+from ibm_cloud_sdk_core.authenticators import IAMAuthenticator
+from ibm_cloud_sdk_core.api_exception import ApiException
+from jproperties import Properties
+import pytest
+
+# Read config file
+configFile = 'global_catalog.env'
+config = {}
+configLoaded = None
+
+try:
+    with open(configFile, "rb") as f:
+        p = Properties()
+        p.load(f, "utf-8")
+        config['GLOBAL_CATALOG_API_URL'] = p['GLOBAL_CATALOG_API_URL'].data
+        config['GLOBAL_CATALOG_API_KEY'] = p['GLOBAL_CATALOG_API_KEY'].data
+        config['GLOBAL_CATALOG_IAM_URL'] = p['GLOBAL_CATALOG_IAM_URL'].data
+        configLoaded = True
+except:
+    print('External configuration was not found, skipping tests...')
+
+
+class TestGlobalCatalogV1(unittest.TestCase):
+
+    def setUp(self):
+        if not configLoaded:
+          self.skipTest("External configuration not available, skipping...")
+
+        apiKey = config['GLOBAL_CATALOG_API_KEY']
```

Comment:
Rather than load up the config file and deal with the individual properties yourself, you could take advantage of some additional functionality in the python SDK and core libraries to use the builtin external config support.
You can see this integration test for an example:
https://github.com/IBM/platform-services-python-sdk/blob/add-iag/test/integration/test_iam_access_groups_v2.py

Basically, just set the IBM_CREDENTIALS_FILE env var to point to your .env file, then just call the service's `new_instance()` method.   This method will load the config file and use the properties to construct an authenticator, then construct an instance of the service.
To leverage this, your config file would need to be changed slightly to use different property names:
```
GLOBAL_CATALOG_AUTHTYPE=iam
GLOBAL_CATALOG_APIKEY=my-iam-apikey
GLOBAL_CATALOG_AUTH_URL=https://iam.cloud.ibm.com/identity/token
GLOBAL_CATALOG_URL=my-service-url
```
Also, you can omit the AUTH_URL property if you're just using the default IAM endpoint.

One additional note...  if your integration test also needs to retrieve one or more test-specific properties (perhaps parameter values that you use in your test), the example above shows how to do that as well (look for the "TEST_ACCOUNT_ID" property name).